lib/elixir/pages/Writing Documentation.md

```diff
@@ -53,33 +53,41 @@ The compiler will infer this argument as `map`. Sometimes the inference will be
 
 ## Recommendations
 
-There are a couple tips we recommend developers to follow when writing documentation:
+When writing documentation:
```

Comment:
I respectfully disagree. The original sentence is "fluffy". I prefer getting straight to the point.